api/ByteLengthQueuingStrategy.json

```diff
@@ -125,10 +125,10 @@
               "version_added": "43"
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
This looks wrong, I don't think this API makes any sense without a constructor.

https://mdn-bcd-collector.appspot.com/tests/api/ByteLengthQueuingStrategy/ByteLengthQueuingStrategy returns null on Safari 14, so is this a bug in update-bcd treating null as false?

Reply:
I've filed https://github.com/foolip/mdn-bcd-collector/issues/883 about this.

Reply:
Ah, yeah, this is an `update-bcd` bug.  Lemme fix this data manually.

Reply:
Thanks!